### ISSUE TITLE
Chapter resumes at previous page while in dual page layout mode

### ISSUE THREAD

**Issue by curche** [jobobby04/TachiyomiSY#935]
### Steps to reproduce

1. Open a chapter
2. Read it in dual page layout
3. Read a few pages, note the page that you stopped at and exit the reader.
4. Observe the page number recorded at the chapter list
5. Open the chapter again and observe what page it resumes at.

### Expected behavior

The reader resumes the chapter at the page where you left it

### Actual behavior

The reader resumes the chapter at few pages before the page where you left it at.

### Tachiyomi version

528

### Android version

13

### Device

Samsung Galaxy Note 10 Lite

### Other details

Its usually 1-2 pages before the current page and I noticed it mostly while reading and resuming with dual page layout turned on

### Acknowledgements

- [X] This is a TachiyomiSY specific issue that does not happen in [Tachiyomi Preview](https://github.com/tachiyomiorg/tachiyomi/).
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.9.3](https://github.com/jobobby04/tachiyomisy/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by DatHypnoboi** [comment]
Yeah, dual page layout is pretty janky in general, another issue is if you toggle from single page to dual page or vise versa while reading a chapter, it goes back a few pages. Seemingly, dual page layout doesn't properly register the currently viewed page
